tests/rules/tcp-seq-keyword/test.yaml

```diff
@@ -0,0 +1,21 @@
+requires:
+    min-version: 7.0.0
```

Comment:
Just confirmed: none of these tasks to add details to `engine-analysis` output will be available for 7, so all PR for SV tests related to similar changes should have `min-version: 8.0`

Reply:
OK! Understood.

Thank you.